src/fluent-plugin-mdsd/lib/fluent/plugin/out_mdsd.rb

```diff
@@ -19,6 +19,10 @@ def initialize()
         config_param :djsonsocket, :string
         desc 'if no ack is received from mdsd after N milli-seconds, drop msg.'
         config_param :acktimeoutms, :integer
+        desc 'Fluentd tag prefix that will be treated as mdsd syslog messages'
+        config_param :mdsd_syslog_tag_prefix, :string, default: nil
+        desc 'If true, change (unify) any tags starting with the prefix to the prefix only'
+        config_param :unify_mdsd_syslog_tags, :bool, default: false
```

Comment:
I don't think that we need 'unify_mdsd_syslog_tags' . we can change the logic such that: if mdsd_syslog_tag_prefix is nil, then don't unify mdsd syslog; if mdsd_syslog_tag_prefix is not nil, unify mdsd syslog tag using that prefix.